presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataReader.java

```diff
@@ -391,6 +393,21 @@ private static DateStatistics toDateStatistics(HiveWriterVersion hiveWriterVersi
                 dateStatistics.hasMaximum() ? dateStatistics.getMaximum() : null);
     }
 
+    private static TimestampStatistics toTimestampStatistics(HiveWriterVersion hiveWriterVersion, OrcProto.TimestampStatistics timestampStatistics, boolean isRowGroup)
+    {
+        if (hiveWriterVersion == ORIGINAL && !isRowGroup) {
```

Comment:
What I mean is, we should have a comment linking to the issue stating that it was broken and is fixed after version `xxx`.  As for the specific issue, it wasn't clear to me in which version they fixed the issue.